wordpress-mobile/WordPress-iOS
# Issue/8377 mobile themes endpoint

Fixes #8377 

Needs Review: @nheagy

To test:
- Start the Site Creation process. Select a Category.
- Ensure the Themes displayed match the selected category. (Related information at p77Llu-9vo-p2)

Blog:
- Intergalactic 2
- Independent Publisher 2
- Penscratch 2
- Libre 2

Website:
- Radcliffe 2
- Twenty Seventeen
- Dara
- Karuna

Portfolio:
- AltoFocus
- Lodestar
- Rebalance
- Sketch
